roboflow/core/project.py

```diff
@@ -101,30 +102,13 @@ def __image_upload(self, image_path, hosted_image=False, split="train"):
             image_name = os.path.basename(image_path)
 
             # Construct URL for local image upload
-
             self.image_upload_url = "".join([
                 "https://api.roboflow.com/dataset/", project_name, "/upload",
-                "?api_key=", self.__api_key,
-                "&name=" + image_name,
-                "&split=" + split
+                "?api_key=", self.__api_key
             ])
 
-            # Convert to PIL Image
-            img = cv2.imread(image_path)
```

Comment:
wondering if we may be changing (undocumented?) behavior here:

Looks like before we were reading the image with PIL and then "saving" to JPEG before base64 encoding; whereas now we just upload the image directly as is on the local filesystem.

I don't think our API handles conversion, so may be safer to leave the PIL loading and JPEG conversion in?  I could imagine e.g. a customer currently using this to upload a folder of png, tiff or any other PIL supported image format and then it just magically working and then breaking their workflow by changing this to upload the file without converting to JPEG.

Reply:
@yeldarby are the images we save to GCS always jpeg?

Reply:
Not sure; what do we do in the frontend? I think we upload them as-is & convert them as part of `roboflow-transform`.

We should verify that they are images for sure though; don't want someone uploading a malicious exe & serving it from our bucket or something.

Reply:
Good catch, so we want to leave the PIL loading and JPEG conversion and then send that JPEG in the multipart form?

Reply:
> so we want to leave the PIL loading and JPEG conversion and then send that JPEG in the multipart form
yes, I think that would be ideal